src/pls/models/node.py

```diff
@@ -243,8 +244,9 @@ def table_row(self) -> Union[dict[str, str], None]:
         if args.details:
             cells["type"] = self.type_char
             cells["perms"] = get_permission_text(self.stat.st_mode)
```

Comment:
Does the permission text column work on Windows? I expected that would be something that only applies to POSIX?

Reply:
Seems to
![pls-details-windows](https://user-images.githubusercontent.com/37764578/153699730-4b916a70-ba95-4960-8e2f-ecd012a3ecf3.png)

Reply:
That's very cool! 

I don't have a windows machine to test this out. Thanks for sharing!